src/cryptography/hazmat/backends/commoncrypto/ciphers.py

```diff
@@ -8,6 +8,7 @@
 from cryptography.exceptions import (
     InvalidTag, UnsupportedAlgorithm, _Reasons
 )
+from cryptography.hazmat.backends.utils import GcmSizeValidator
```

Comment:
Opps, I read the style guidelines but flipped a bit in my head for "A class should be named HTTPClient, not HttpClient."